packages/redux/README.md

```diff
@@ -55,6 +66,36 @@ ReactDOM.render(Start, document.getElementById('root'));
 registerServiceWorker();
 ```
 
+The optional parameter "isEnabled" allows you to enable or disable authentication. You will also find it in the `OidcSecure` component.
+
+"Authentificationprovider" accept the following properties :
+ ```javascript
+const propTypes = {
+  notAuthentified: PropTypes.node, // react component displayed during authentication
+  notAuthorized: PropTypes.node, // react component displayed in case user is not Authorised
+  configuration: PropTypes.shape({
+    client_id: PropTypes.string.isRequired, // oidc client configuration, the same as oidc client library used internally https://github.com/IdentityModel/oidc-client-js
+    redirect_uri: PropTypes.string.isRequired,
+    response_type: PropTypes.string.isRequired,
+    scope: PropTypes.string.isRequired,
+    authority: PropTypes.string.isRequired,
+    silent_redirect_uri: PropTypes.string.isRequired,
+    automaticSilentRenew: PropTypes.bool.isRequired,
+    loadUserInfo: PropTypes.bool.isRequired,
+    triggerAuthFlow: PropTypes.bool.isRequired
+  }).isRequired,
+  isEnabled: PropTypes.bool, // enable/disable the protections and trigger of authentication (useful during development).
+  loggerLevel: PropTypes.number,
+  logger: PropTypes.shape({
```

Comment:
These parameters are not available for "oidc-redux"

Reply:
changed, thanks for review